.github/workflows/e2e_pr.yml

```diff
@@ -44,13 +44,11 @@ jobs:
           DATABASE_URL: postgres://postgres:postgres@localhost:5432/blurts
       - name: Store Playwright's Version
         run: |
-          # The regular expression matches everything containing
-          # `<number>.<number>.<number>`, and returns only that bit:
-          PLAYWRIGHT_VERSION=$(npm ls @playwright/test | grep @playwright | sed --regexp-extended 's/.*([0-9]+\.[0-9]+\.[0-9]+).*/\1/') | head -1
-          echo "Playwright's Version: $PLAYWRIGHT_VERSION"
+          # Get the current Playwright version listed in package.json
+          PLAYWRIGHT_VERSION=$(npm view @playwright/test version)
+          echo "Playwright Version: $PLAYWRIGHT_VERSION"
```

Comment:
@mansaj It seems like there was an issue with determining the current Playwright version in CI. We should be able to just directly get it with `npm view @playwright/test version`.

Reply:
Yeah, that’s right: `npm view` might be a potential mismatch because it checks the remote version. Otherwise, `npm list @playwright/test --depth=0 | grep @playwright/test | sed 's/.*@//'` could work as well. I like your suggestion to use `npx` since it is easy to parse — added in [a8bdde6](https://github.com/mozilla/blurts-server/pull/4585/commits/a8bdde6a9a4238c6b42142121b493302181c1178).